README.md

```diff
@@ -0,0 +1,16 @@
+# docker-fsfreeze
+
+[![CircleCI](https://circleci.com/gh/Intellection/docker-fsfreeze/tree/master.svg?style=svg&circle-token=61fff2811f0766a50f34853754ccd75bbe2d4ffd)](https://circleci.com/gh/Intellection/docker-fsfreeze/tree/master)
+
+The [`fsfreeze`][1] command is used to suspend and resume access to a file system. This allows consistent snapshots to be taken by halting any new write system calls while allowing ongoing transactions to complete.
+
+Supported filesystems:
+* Ext3
+* Ext4
+* ReiserFS
+* JFS
+* XFS
+
+Note: `fsfreeze` is unncessary for device-mapper devices. The device-mapper (and LVM) automatically freezes filesystem on the device when a snapshot creation is requested.
```

Comment:
*unnecessary

Reply:
Hah, I copied it from the [man page](https://linux.die.net/man/8/fsfreeze).

Reply:
Addressed in https://github.com/Intellection/docker-fsfreeze/pull/1/commits/0572d7cff2db4dd269feb835ff4bec47753e669f.

Reply:
Haha
Whoops